tests/unit/feeds/new/add-operator/controller-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('controller:feeds/new/add-operator', 'Unit | Controller | feeds/new/add operator', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
+});
+
+// Replace this with your real tests.
+test('it exists', function(assert) {
+  let controller = this.subject();
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).